schemas/string_interpolation/transforms/format_unit.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/format_unit.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "format_unit": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["format_unit"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/placeholder_value_sources"
+          },
```

Comment:
I think we should add some validation to ensure if an answer value source is used, then that answer's type matches the `unit`, otherwise we should raise a validation error.

Reply:
Ok, so I added new validation to validate exactly this.